### ISSUE TITLE
Bring OpenDroneMap in SciF up to date and test

### ISSUE THREAD

**Issue by Chris-Schnaufer** [AgPipeline/drone-pipeline-environment#80]
**Task to do**
Ensure ODM as a separate app is available with SciF solution

**Reason**
Running ODM to generate an OrthoMosaic is needed

**Result**
ODM is ensured to be available

**Steps to take**
- [x] Make sure DPP ODM Docker image is up to date
- [x] Confirm integration with SciF solution
- [x] Test SciF ODM app

**Comment by Chris-Schnaufer** [comment]
Perhaps an approach is to use deadsnakes' python3.7 install in the ODM docker image as the basis for the SciF system (instead of Ubuntu 18.04)

**Comment by Chris-Schnaufer** [comment]
The PR is in another issue since there's a previous PR that this change is waiting on.

Issue: https://github.com/AgPipeline/issues-and-projects/issues/183
